addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -292,21 +291,27 @@ odoo.define('website_slides.quiz', function (require) {
                     quizKarmaGain: slideData.quizKarmaGain,
                     quizAttemptsCount: slideData.quizAttemptsCount,
                 };
-                defs.push(new Quiz(self, slideData, slideData.quizData).attachTo($(this)));
+                var quiz = new Quiz(self, slideData, slideData.quizData)
+                self.quizWidgets.push(quiz);
+                defs.push(quiz.attachTo($(this)));
             });
             return $.when.apply($, defs);
         },
 
         //----------------------------------------------------------------------
         // Handlers
         //---------------------------------------------------------------------
-
-        _onQuizCompleted: function (slideId) {
-            console.log('set completed', slideId);
+        _onQuizCompleted: function (ev) {
+            var self = this;
+            var slide = ev.data.slide;
+            var completion = ev.data.completion;
+            this.$('#o_wslides_lesson_aside_slide_check_' + slide.id).addClass('text-success fa-check-circle').removeClass('text-600 fa-circle-o');
+            // need to use global selector as progress bar is ouside this animation widget scope
```

Comment:
Then it should probably trigger_up an event that is caught by the parent.

Reply:
But there is not parent, or we will have to create a widget for all the page O_0 So, tampis for now.